lib/request.coffee

```diff
@@ -83,15 +83,15 @@ module.exports = getRequest = ({
 					code: 'ResinRequestError'
 					statusCode: 401
 				, ->
-					return token.get().tap(token.remove).then (sessionToken) ->
+					return auth.getKey().tap(auth.removeKey.bind(auth)).then (sessionToken) ->
```

Comment:
@MoranF I think the final conclusion of https://github.com/resin-io-modules/resin-request/pull/107#discussion_r147462011 was actually that you _don't_ need the `.bind` here. You can add it anyway if you like, but it's actually not necessary, we were wrong.